### DISCUSSION TITLE
Customize Image Column

### DISCUSSION THREAD

**Discussion by ayemin-tun** [filamentphp/filament#9784] [General]
### Description

Can I add border in image column? I use extraImageAttribute but not work.

**Comment by coolsam726** [comment]
Use `->extraImgAttributes()`. The following works fine for me:
```php
Tables\Columns\ImageColumn::make('image')->extraImgAttributes([
                    'class' => 'border-2 border-primary-500'
                ])->getStateUsing(fn() => 'https://placehold.co/600x400/png'),
```
![image](https://github.com/filamentphp/filament/assets/5610289/adfb945b-0489-418b-98f9-3260be181407)

**Comment by Khant-Nyar** [comment]
i save actor as name and image array cast with $table->json('actors')

how to display image like that 
![image](https://github.com/filamentphp/filament/assets/61083917/7d798e84-f818-4dbc-9ffb-0f67ccf2169b)

**Reply by Khant-Nyar** [reply]
solved it 

```
Tables\Columns\ImageColumn::make('actors')
                    ->label('Actors')
                    ->getStateUsing(function (Movie $record) {
                        $profileUrls = collect($record->actors)->pluck('profile')->toArray();

                        return $profileUrls;
                    })
                    ->circular()
                    ->stacked()
                    ->limit(3)
                    ->limitedRemainingText(),
```

![image](https://github.com/filamentphp/filament/assets/61083917/75250179-6b30-4339-8da6-5caad2141114)

**Reply by AliKar97** [reply]
thanks for sharing, bro
